development.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```

Comment:
@jone, I don't remember if you told me once not to switch the default Plone versions or somebody else. Anyway, what's your opinion on this?

Reply:
I believe we solved the issue in this package yesterday. I'll go on here, when ftw.simplelayout is newly released and comment on the issue again.

Reply:
@mbaechtold @busykoala at the moment we are mostly using the package with Plone 4. So if someone checks it out and makes a change, the change is most likely for a Plone 4 project. So I personally would prefer that we keep extending `4.3.x.cfg` in `development.cfg` for convenience. 
Once the active development in the projects is more Plone 5 than 4 we can start switching it.

Reply:
👍

Reply:
Thanks @jone 👍